src/main/server/start.sh

```diff
@@ -45,6 +45,15 @@ DB_USERNAME=
 DB_PASSWORD=
 LICENSE_KEY=
 
+# Set the default value for JVM_MEMORY_OPTIONS
+DEFAULT_MIN_HEAP_SIZE="1g"
+JVM_MEMORY_OPTIONS="${JVM_MEMORY_OPTIONS:="-Xms$DEFAULT_MIN_HEAP_SIZE"}"
+
+# If the initial Java heap size (-Xms) and the maximum heap size (-Xmx) values are specified as environment variables (ENV) in the Dockerfile, those values will be set.
```

Comment:
How the environment variable for -Xms and -Xmx are set to the JVM options?

Reply:
Hi @mekya 

I've already added it as a parameter. FYI.

https://github.com/ant-media/Ant-Media-Server/pull/5342/files#diff-a4cc622618e697532f134dfe01c55653ce830c08217f7d430bf80ae15e3297f2R69